### PR TITLE
Added Avatars 

### PR DESCRIPTION
**Pull request by atharvabhanage02**

### CODE REVIEW

File: components/avatars/avatar.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.avatar-round-sm{
+    width: 2rem;
+    height: 2rem;
+}
+.avatar-round-md{
+    width: 4rem;
+    height: 4rem;
+}
+.avatar-round-lg{
+    width: 6rem;
```
Comment (Raghav888) [inline components/avatars/avatar.css:14]: If possible use var here, that will help you in future

**Review by Raghav888 — APPROVED** [review]
Good to merge

**Review by sks-21 — APPROVED** [review]
Good use of comments.

File: components/avatars/avatar.css (hunk 1 above)
Comment (atharvabhanage02) [reply components/avatars/avatar.css:14]: Yes. I'll keep this is mind.
Thanks for the suggestion.

**Comment by atharvabhanage02** [comment]
Thank You Shashank

**Pull request merged**